src/components/axes/timeAxis.ts

```diff
@@ -240,8 +244,13 @@ export module Axis {
 
     public _computeHeight() {
       var textHeight = this._measurer.measure().height;
-      this._tierHeights = this._tierLabelPositions.map((pos: string) =>
-        textHeight + this.tickLabelPadding() + ((pos === "between") ? 0 : this._maxLabelTickLength()));
+
+      this._tierHeights = [];
```

Comment:
Hmm, I guess we don't use the number of tiers anywhere else. This might be a good change to make.